python/private/bzlmod/pip.bzl

```diff
@@ -25,12 +25,29 @@ load(
     "whl_library",
 )
 load("//python/pip_install:requirements_parser.bzl", parse_requirements = "parse")
-load("//python/private:full_version.bzl", "full_version")
 load("//python/private:normalize_name.bzl", "normalize_name")
 load("//python/private:parse_whl_name.bzl", "parse_whl_name")
 load("//python/private:version_label.bzl", "version_label")
 load(":pip_repository.bzl", "pip_repository")
 
+def _parse_version(version):
+    major, _, version = version.partition(".")
+    minor, _, version = version.partition(".")
+    patch, _, version = version.partition(".")
+    build, _, version = version.partition(".")
```

Comment:
(no action required)

Is the 4-dot component notation pip or semver?

IIRC, the python sys.version format is "3.10.5a4" (3 dots, letter, serial).

I don't think it matters much for now as we only ever have "x.y.z" for the python versions